### ISSUE TITLE
Add Support for Spain

### ISSUE THREAD

**Issue by eprbell** [eprbell/rp2#61]

**Comment by Larision** [comment]
Hi, i´m trying (i´m not a programmer) add support to spain, i´ve country plugin, translations and using full_report, but i have a problem with templates in dali-rp2 showing fiat in $. template.ods not country-specific and i dont know how to solve it. any tip? Thanks

**Comment by macanudo527** [comment]
The template is country-specific. You'll have to make a new one for Spain in order for the currency to show in Euros. If you submit a PR, we could probably walk you through it.

**Comment by eprbell** [comment]
> The template is country-specific. You'll have to make a new one for Spain in order for the currency to show in Euros. If you submit a PR, we could probably walk you through it.

Yes, and you can also look at the plugin for Japan, which already went through the procedure. Here are a few relevant bits:
* [Japanese templates](https://github.com/eprbell/rp2/tree/main/src/rp2/plugin/report/data/jp),
* [Japanese report referring to said templates](https://github.com/eprbell/rp2/blob/main/src/rp2/plugin/report/jp/tax_report_jp.py#L105),
* [Japanese translation](https://github.com/eprbell/rp2/tree/main/src/rp2/locales/ja/LC_MESSAGES) (they're currently absent but they would go where the empty strings are).
